ingest-app/docs/api.md

```diff
@@ -622,7 +622,7 @@ Example output:
 ```
 ## <a name="translate-granule"></a> Translate Granule Metadata
 
-Granule metadata can be translated between metadata standards using the translate API in Ingest. The request specifies the metadata standard being sent using the Content-Type header. Metadata is sent inside the body of the request. The output format is specified via the Accept header. The supported formats are ECHO10, ISO SMAP and UMM-G.
+Granule metadata can be translated between metadata standards using the translate API in Ingest. The request specifies the metadata standard being sent using the Content-Type header. Metadata is sent inside the body of the request. The output format is specified via the Accept header. The supported input formats are ECHO10, ISO SMAP and UMM-G. The supported output formats are ECHO10, ISO SMAP, UMM-G and ISO19115. 
```

Comment:
Why is ISO 19115 not supported as an input format? Do we support all combinations of input and output format? For example can we use ISO SMAP as input and get back ISO-19115?

Reply:
We don't have support in either umm-lib or umm-spec-lib to parse ISO19115 granules. That is why we can not ingest granule in ISO19115 metadata format. ISO19115 is strictly an output format for granules. We can get granule in ISO19115 from any valid granule formats including ISO SMAP.